### PR TITLE
give the new world the good old names

### PR DESCRIPTION
**Pull request by thomas-fossati**
do the heavy lifting

### CODE REVIEW

File: concise-mid-tag.cddl [hunk 1]
```diff
@@ -1,136 +1,116 @@
...
+  ? comid.tag-version => tag-version-type
 }
 
 $tag-id-type-choice /= tstr
 $tag-id-type-choice /= uuid-type
 
 tag-version-type = uint .default 0
 
-module-entity-map = {
+entity-map = {
```
Comment (yogeshbdeshpande) [inline concise-mid-tag.cddl:22]: Please remove module word from role-type-choice also...

File: concise-mid-tag.cddl [hunk 2]
```diff
@@ -1,136 +1,116 @@
...
   comid.entity-name => $entity-name-type-choice
   ? comid.reg-id => uri
   comid.role => one-or-more<$module-role-type-choice>
-  * $$module-entity-map-extension
+  * $$entity-map-extension
```
Comment (yogeshbdeshpande) [inline concise-mid-tag.cddl:26]: Between, where would the new role-type-choice been housed ??

File: comid-code-points.cddl [hunk 3]
```diff
@@ -19,101 +18,52 @@ comid.role = 2
...
-comid.layer = 4
-comid.index = 5
+comid.model = 2
+comid.layer = 3
+comid.index = 4
 
-; element-value-map
-comid.version = 0
+; measurement-map
+comid.ver = 0
```
Comment (yogeshbdeshpande) [inline comid-code-points.cddl:40]: Perhaps a comment on clarifying what does the comid.ver implies as against comid.version below, might help to a reader!

File: concise-mid-tag.cddl (hunk 1 above)
Comment (thomas-fossati) [reply concise-mid-tag.cddl:22]: mmm why?  we use a similar convention for corim and xcorim roles.  Maybe we could change the prefix from `module` to `comid`, but I don't understand why we should remove it altogether

File: concise-mid-tag.cddl (hunk 2 above)
Comment (thomas-fossati) [reply concise-mid-tag.cddl:26]: good catch!

File: comid-code-points.cddl (hunk 3 above)
Comment (thomas-fossati) [reply comid-code-points.cddl:40]: I expect the text in the draft to explain such things.  Besides the comments are removed when the final CDDL is coalesced.

File: concise-mid-tag.cddl (hunk 2 above)
Comment (thomas-fossati) [reply concise-mid-tag.cddl:26]: fixed in e77668c

File: concise-mid-tag.cddl (hunk 1 above)
Comment (thomas-fossati) [reply concise-mid-tag.cddl:22]: changed `module` into `comid` e77668c

Comment (yogeshbdeshpande) [reply concise-mid-tag.cddl:22]: Works for me!

**Review by yogeshbdeshpande — APPROVED** [review]

**Pull request merged**